e2e/test-component/scenarios/embedding-sdk/interactive-question-theming.cy.spec.tsx

```diff
@@ -48,79 +48,65 @@ describe(
     });
 
     it("derives dynamic css variables for dark theme", () => {
-      const theme: MetabaseTheme = {
+      const BACKGROUND_COLOR = "rgb(22, 26, 29)";
+
+      setupInteractiveQuestionWithTheme({
         colors: {
-          background: "rgb(22, 26, 29)",
+          background: BACKGROUND_COLOR,
           "background-hover": "rgb(14, 17, 20)",
           "background-disabled": "rgb(45, 45, 48)",
           "text-primary": "rgb(255, 255, 255)",
           brand: "rgb(253, 121, 168)",
         },
-      };
-
-      cy.get<number>("@questionId").then((questionId) => {
-        mountSdkContent(
-          <Box bg={theme.colors?.background} h="100vh">
-            <InteractiveQuestion questionId={questionId} />
-          </Box>,
-          { theme },
-        );
       });
 
       getSdkRoot().within(() => {
         cy.findByText("Product ID").should("be.visible");
 
-        const buttonHoverBg = lighten(theme.colors?.background, 0.5);
+        const buttonHoverBg = lighten(BACKGROUND_COLOR, 0.5);
 
         const customColumn = "[aria-label='Custom column']";
 
         // Should be the lightened version of the background color
         cy.findByTestId("notebook-button")
           .should("be.visible")
           .realHover()
-          .should(($el) => haveBackgroundColor($el, buttonHoverBg));
+          .should(($el) => assertBackgroundColorEqual($el, buttonHoverBg));
```

Comment:
The previous `haveBackgroundColor` did not do any assertion at all (!) so the tests used to always pass. I replaced that with `assertBackgroundColorEqual`, which also have a test for the assertion itself below.